Automattic/wp-calypso
# Use Jetpack.com GA4 Tracking when in Jetpack Checkout

#### Changes proposed in this Pull Request

* Add GA4 purchase event on Jetpack checkouts

#### Testing instructions

1. Modify `development.json` setting the `ad-tracking` feature to true, boot calypso blue
2. Sandbox `public-api.wordpress.com` and use sandbox store ( `PCYsg-IA-p2`)
3. Spin up a fresh jurassic.ninja site, connect to WordPress.com, and wait on the `/pricing` page
4. copy the link address of a product, and modify to use `calypso.localhost:3000` instead of `wordpress.com`
5. open dev tools, make sure the console is being preserved, enter `localStorage.setItem( 'debug', 'calypso:analytics:ad' );`,and  reload page
6. Complete purchase ( keep in mind the store sandbox instructions above )
7.  Verify that `recordOrderInJetpackGA: Record Jetpack Purchase` was logged in the console with params that look correct for the product you bought
